### ISSUE TITLE
removed

### ISSUE THREAD

**Issue by ghost** [Nerodesk/nerodesk#217]

**Comment by yegor256** [comment]
@k0r looks like a duplicate of #216, isn't it?

**Comment by k0r** [comment]
@yegor256 the other ticket is about the login page, whereas this ticket is once you're logged in and view your files.

**Comment by yegor256** [comment]
@k0r but we'll update the font only once and for the entire website. no need to do it twice

**Comment by k0r** [comment]
@yegor256 thought I'd have to make separate tickets for the request.

**Comment by k0r** [comment]
@yegor256 close?

**Comment by yegor256** [comment]
@k0r I would suggest to close it, yes
